src/AzureAppConfigurationImpl.ts

```diff
@@ -657,3 +711,7 @@ function getValidFeatureFlagSelectors(selectors?: SettingSelector[]): SettingSel
         return getValidSelectors(selectors);
     }
 }
+
+function isFailoverableError(error: any): boolean {
+    return isRestError(error) && (error.statusCode === 408 || error.statusCode === 429 || (error.statusCode !== undefined && error.statusCode >= 500));

```

Comment:
Why do you only check whether error.statusCode is undefined for 5xx?

Reply:
`===` operation will check if statusCode is undefined but `>=` not.

Reply:
![image](https://github.com/user-attachments/assets/3211d1e6-0f7a-4e8b-bdd8-563a50ad1e9d)

Reply:
Why not just error.statusCode >= 500?